src/utils/form-actions.ts

```diff
@@ -53,7 +53,7 @@ export async function saveSession(session: Session) {
       operation: 'update',
     });
   }
-  return data;
+  return data as Session;
```

Comment:
why do we need this typecast? i feel like i remember putting in a lot of effort specifically so that we didn't need it

Reply:
Ahhh I can't remember but it was causing some issues somewhere. I'll remove and see what it complains about.

Reply:
Removed, no longer causing an issue anymore.

Reply:
Actually it is an issue but just in the the `server-action.test.ts`

If we don't cast it we have to mimic the SessionStorageModel we are getting back from amplify. So it will become something like this in order to get the mocking to work.

```
      saveSessionMock.mockResolvedValue({
        ...initialState,
        emailTemplateName: 'template-name',
        emailTemplateSubjectLine: 'template-subject-line',
        emailTemplateMessage: 'template-message',
        smsTemplateName: null,
        smsTemplateMessage: null,
        updatedAt: 'today',
        createdAt: 'yesterday',
        ttl: 1000,
      });
  ```
  rather than just
  ```
  saveSessionMock.mockResolvedValue({
        ...initialState,
        emailTemplateName: 'template-name',
        emailTemplateSubjectLine: 'template-subject-line',
        emailTemplateMessage: 'template-message',
      });
  ```
Doing this just get the test to work isn't the right thing to do but I do think casting here is the right thing to do. I don't thing we should be leaking the backend model to the rest of the app. But happy to discuss further.